src/sentry/integrations/metric_alerts.py

```diff
@@ -222,7 +222,8 @@ def incident_attachment_info(
     if notification_uuid:
         title_link_params["notification_uuid"] = notification_uuid
 
-    if should_fire_workflow_actions(organization):
+    # TODO(iamrajjoshi): This will need to be updated once we plan out Metric Alerts rollout
+    if should_fire_workflow_actions(organization, 8001):
```

Comment:
use MetricIssue.type_id? need to import it

```suggestion
    if should_fire_workflow_actions(organization, `MetricIssue.type_id`):
```